documentation/asciidoc/accessories/iqaudio/digiamp_plus.adoc

```diff
@@ -1,22 +1,22 @@
 === Raspberry Pi DigiAMP{plus}
 
-With Raspberry Pi DigiAMP{plus}, you can connect passive stereo speakers up to 2x35W with variable output, making it ideal for use for Raspberry Pi based Hi-Fi systems.
+With Raspberry Pi DigiAMP{plus}, you can connect passive stereo speakers up to 2x35W with variable output, making it ideal for use in Raspberry Pi-based hi-fi systems.
 
-DigiAMP{plus} uses the Texas Instruments TAS5756M PowerDAC and must be powered from an external supply. It requires a 12-24V DC power source (XP Power VEC65US19 power supply is recommended).
+DigiAMP{plus} uses the Texas Instruments TAS5756M PowerDAC and must be powered from an external supply. It requires a 12-24V DC power source (the XP Power VEC65US19 power supply is recommended).
 
 image::images/DigiAMP+_Board_Diagram.jpg[width="80%"]
 
 DigiAMP{plus}’s power in barrel connector is 5.5mm x 2.5mm. 
 
 At power-on, the amplifier is muted by default (the mute LED is illuminated). Software is responsible for the mute state and LED control (Raspberry Pi GPIO22).
 
-DigiAMP{plus} is designed to provide power to the Raspberry Pi and DigiAMP{plus} together in parallel, delivering 5.1V at 2.5Amp to the Raspberry Pi through the GPIO header.
+DigiAMP{plus} is designed to provide power to the Raspberry Pi and DigiAMP{plus} together in parallel, delivering 5.1V at 2.5amp to the Raspberry Pi through the GPIO header.
```

Comment:
2.5amp -> 2.5A ? :shrug:

Reply:
Liz you can't decapitalise Amps, and Amp is an Amp, it is not an amp. It is (a) the name of the person, and (b) the name of the unit. Units named after people are always uppercased in physics. Should be "2.5 A" or "2.5 Amps"

Reply:
You left volts capitalised, so it's "5.1 V" not "5.1 v". 

That said engineers have a habit of lowercasing volts, and I will hunt all instances out and fix them. It's `V` not `v`, and `A` not `a`.

Reply:
And [the style-guide](https://github.com/raspberrypi/style-guide/blob/master/style-guide.md) also says no space between the number and the unit, so "5.1V" not "5.1 V"

Reply:
*mutters*

Reply:
`Amp` and `amp` are equally wrong - it should be `A`.